<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PumpOperateDialog</class>
 <widget class="QDialog" name="PumpOperateDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Pump operate</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 8pt &quot;Arial&quot;;</string>
  </property>
  <widget class="QPushButton" name="Button_man">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>42</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>MAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_off">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>42</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_auto">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>42</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>AUTO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>14</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_trip">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>TRIP</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>65</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="DoubleInputEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleInputEdit</class>
   <extends>QLineEdit</extends>
   <header>doubleinputedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
